facilitator/src/transport/gcs.rs

```diff
@@ -215,8 +225,15 @@ impl GCSTransport {
 }
 
 impl Transport for GCSTransport {
+    fn path(&self) -> String {
+        self.path.to_string()
+    }
+
     fn get(&mut self, key: &str) -> Result<Box<dyn Read>> {
-        info!("get {} as {:?}", self.path, self.oauth_token_provider);
+        info!(
+            "get {}{} as {:?}",
```

Comment:
nit: `get {}/{} as {:?}`, unless `self.path` is already guaranteed to be /-terminated.